### ISSUE TITLE
Suggest add viewport meta tag for mobile.

### ISSUE THREAD

**Issue by mixed** [jquery/qunit#913]
Result page is hard to read in mobile. Adding meta tag looks good.
```html
<meta name="viewport" content="width=device-width">
```
## sample
![photo_0](https://cloud.githubusercontent.com/assets/22300/12048301/bebe2f96-af1c-11e5-9d2f-2df77cf33da8.png)

## current
![photo_1](https://cloud.githubusercontent.com/assets/22300/12048302/c30a2b5e-af1c-11e5-8ebb-83069d0b45c6.png)

**Comment by JamesMGreene** [comment]
:+1:

**Comment by JamesMGreene** [comment]
So, typically the consumer is the one who provides the HTML page on which the unit tests are run. Are you just suggesting that it should be included on a sample page somewhere, like the one on qunitjs.com?

**Comment by mixed** [comment]
@JamesMGreene 
Failing my [pr](https://travis-ci.org/jquery/qunit/builds/99386558) I checked real browser in iOS. Finding it I suggested. So I thought that It add to all html in `/test` folder. But Adding to sample in qunit.js It seems to good since I saw your comment.

**Comment by gibson042** [comment]
We also might look into adding something to our CSS (media queries, viewport units, etc.).

**Comment by leobalter** [comment]
@mixed You might want to include this here: https://github.com/jquery/qunitjs.com/blob/master/resources/example-index.html

The many people use the html example on qunitjs.com. We may add it here on QUnit's test html files but it will affect only those running QUnit's own tests.

> @gibson042: We also might look into adding something to our CSS (media queries, viewport units, etc.).

That would be interesting too.

**Comment by mixed** [comment]
@leobalter 
Thank you. Can I change `example-index.html`?

**Comment by leobalter** [comment]
sure, but it's on another repo: https://github.com/jquery/qunitjs.com

**Comment by mixed** [comment]
@leobalter 
OK. Done. https://github.com/jquery/qunitjs.com/pull/110

**Comment by leobalter** [comment]
@mixed, do you want to add a PR to add the viewport tag on QUnit's html files? So we may solve this issue.

**Comment by mixed** [comment]
No. I found your [comments](https://github.com/jquery/qunit/issues/913#issuecomment-168022675) so I think that https://github.com/jquery/qunitjs.com/pull/110 sufficed for this issue.
Thank you.

**Comment by leobalter** [comment]
Thanks! 

I'm looking forward to know if you wanna help on any other features/issues.
